package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metarhia/common",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
You can't make it a patch version since there is new functionality added since the last release.
Besides that, `CHANGELOG.md` wasn't updated to reflect the changes, and overall, I think it would be better to follow a standardized release procedure for this repository by making a separate PR from the `publish` branch with a [`release`](https://github.com/metarhia/common/pulls?q=is%3Apr+label%3Arelease+is%3Aclosed) tag and description of the changes in the PR description extracted from the changelog, instead of silently putting out new version inside a PR containing multiple small changes.